src/NServiceBus.Core/Licensing/LicenseExpiredForm.cs

```diff
@@ -1,145 +0,0 @@
-            }
-
-            Visible = true;
-        }
-
-        void browseButton_Click(object sender, EventArgs e)
-        {
-            using (var openDialog = new OpenFileDialog())
-            {
-                openDialog.InitializeLifetimeService();
-                openDialog.Filter = "License files (*.xml)|*.xml|All files (*.*)|*.*";
-                openDialog.Title = "Select License file";
-
-                var dialogResult = StaThreadRunner.ShowDialogInSTA(openDialog.ShowDialog);
-                if (dialogResult == DialogResult.OK)
-                {
-                    var licenseText = NonBlockingReader.ReadAllTextWithoutLocking(openDialog.FileName);
-                    try
-                    {
-                        LicenseVerifier.Verify(licenseText);
-                        var license = LicenseDeserializer.Deserialize(licenseText);
-
-                        if (LicenseExpirationChecker.HasLicenseExpired(license))
-                        {
-                            var message = "The license you provided has expired, select another file.";
-                            Logger.Warn(message);
-                            MessageBox.Show(this, message, "License expired", MessageBoxButtons.OK, MessageBoxIcon.Exclamation);
-                            return;
-                        }
-                        ResultingLicenseText = licenseText;
-                        Close();
-                    }
-                    catch (Exception exception)
-                    {
-                        var message = $"An error occurred when parsing the license.\r\nMessage: {exception.Message}\r\nThe exception details have been appended to the log.";
-                        Logger.Warn("Error parsing license", exception);
-                        MessageBox.Show(this, message, "Error parsing license", MessageBoxButtons.OK, MessageBoxIcon.Error);
-                    }
-                }
-            }
-        }
-
-        void PurchaseButton_Click(object sender, EventArgs e)
-        {
-            Process.Start("https://particular.net/licensing");
-        }
-
-        void getTrialLicenseButton_Click(object sender, EventArgs e)
-        {
-            string baseUrl;
-            if (CurrentLicense != null && !CurrentLicense.IsExtendedTrial)
-            {
-                // Original 14 day trial expired, give the user a chance to extend trial
-                baseUrl = "https://particular.net/extend-nservicebus-trial";
-            }
-            else
-            {
-                // Extended trial license expired, ask the user to Contact Sales
-                baseUrl = "https://particular.net/extend-your-trial-45";
-            }
-
-            var trialStart = TrialStartDateStore.GetTrialStartDate().ToString("yyyy-MM-dd", CultureInfo.InvariantCulture);
-            var url = $"{baseUrl}?NugetUser={IsNugetUser()}&PlatformInstaller={HasUserInstalledPlatform()}&TrialStartDate={trialStart}";
```

Comment:
From what I understand, the init.ps1 will be still be run if the project is being opened with Visual Studio, but I don't think it will be anywhere else.